styles/styles.js

```diff
@@ -0,0 +1,126 @@
+import React, { StyleSheet } from 'react-native'
+
+export default StyleSheet.create({
```

Comment:
instead of exporting your styles as `js` you can directly create a classic CSS file and use classes to set your element style. But you can let the style in the class if you prefer having a single file

Reply:
Do you know if there is a solution better than the other ? or it's a developper choice ?

Reply:
@github-numa-mau, For me if you keep the stylesheet inside the component you've to keep the `StyleSheet.create ` syntax but if you put the styles in an external file, you better use a classic CSS file.

Reply:
@github-thibaud-saintetienne I've put it back in the components, because it seems to be complicated to use normal css files with react natives. So I used the classic Stylesheet in components.